aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -138,6 +139,7 @@ storage:
         inline: |
           #!/bin/bash
           # Wrapper for bootkube start
+          docker pull k8s.gcr.io/hyperkube:v1.17.0
```

Comment:
Please move it after `set -e` and add appropriate comment why it's there.

Reply:
done